datafusion/core/src/datasource/physical_plan/parquet/opener.rs

```diff
@@ -187,15 +190,18 @@ impl FileOpener for ParquetOpener {
             }
             // If there is a predicate that can be evaluated against the metadata
             if let Some(predicate) = predicate.as_ref() {
+                let mut timer = file_metrics.statistics_eval_time.timer();
                 row_groups.prune_by_statistics(
                     &file_schema,
                     builder.parquet_schema(),
                     rg_metadata,
                     predicate,
                     &file_metrics,
                 );
+                timer.stop();
```

Comment:
This one is needed, because it only goes out of scope after the Bloom Filter evaluation. The next `timer.stop()` is indeed redundant, but I kept it for consistency.

Reply:
Doesn't matter anymore, as I moved both calls to the callees.